content/test/integration/denylist/DeactivatedDenylist.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { EnvironmentConfig } from '@katalyst/content/Environment'
+import { assertPromiseIsRejected } from '@katalyst/test-helpers/PromiseAssertions'
+import { loadStandaloneTestEnvironment } from '../E2ETestEnvironment'
+import { buildDeployData, createIdentity } from '../E2ETestUtils'
+import { TestServer } from '../TestServer'
+
+describe('Integration - DeactivatedDenylist', () => {
+  const metadata: string = 'Some metadata'
+  const decentralandIdentity = createIdentity()
+  const testEnv = loadStandaloneTestEnvironment()
+  let server: TestServer
+
+  beforeEach(async () => {
+    server = await testEnv
+      .configServer()
+      .withConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION, true)
+      .withConfig(EnvironmentConfig.DISABLE_DENYLIST, true)
+      .andBuild()
+
+    await server.start()
+  })
+
+  it(`When an entity is denylisted, then an error is thrown`, async () => {
+    // Prepare entity to deploy
+    const { deployData, controllerEntity: entityBeingDeployed } = await buildDeployData(['0,0', '0,1'], {
+      metadata,
+      contentPaths: ['content/test/integration/resources/some-binary-file.png']
```

Comment:
Just a nit. I think you can avoid setting the metadata and the content paths to this entity (and the ones in other tests). It would make the test a little bit more readable